tests/support/loadPV.js

```diff
@@ -0,0 +1,15 @@
+import jsyaml from 'js-yaml';
+
+export async function loadPVFile() {
+  return await new Promise(resolve => {
+    cy.fixture('test-persistent-volumes.yaml').then(fileContent =>
+      resolve(jsyaml.load(fileContent)),
+    );
+  });
+}
+
+export async function loadPV(namespaceName) {
+  // const PV =;
+
+  return await loadPVFile();
+}
```

Comment:
I think we could do with a single function here, instead of two.

Reply:
OK, other loaders should be refactored as well then